pombola/south_africa/static/js/attendance-table.js

```diff
@@ -1,8 +1,13 @@
 jQuery (function($) {
   $(document).ready(function(){
+      var order_by;
+      if (POSITION == 'ministers') {
+        var order_by = [2, "desc"]
```

Comment:
A tiny nitpick, but there `var` on this line isn't necessary, since you've put the `var` at the top of the function anyway, in nice Crockford-style :)